src/Microsoft.DotNet.Darc/Darc/Operations/GatherDropOperation.cs

```diff
@@ -1434,6 +1446,7 @@ private static string GetBlobBaseUri(string blobUri)
                     sourceUri,
                     Logger,
                     authHeader: authHeader,
+                    adjustRequestMessage: AdjustRequestMessage,
```

Comment:
nit: Maybe `configureRequest`? I think that's more usual in ASP.NET terminology?

Reply:
I would agree. But it is a bit confusing in this case because it is not actual configuration of whole message request. It is just a lambda for being able to modify the request just before it is being sent. But it is already configured at that time.

Reply:
I believe that's how ASP.NET usually works too - you just configure what you want on the options object

Reply:
I will update the naming